react/features/toolbox/reducer.ts

```diff
@@ -166,6 +178,12 @@ ReducerRegistry.register<IToolboxState>(
         case SET_TOOLBOX_VISIBLE:
             return set(state, 'visible', action.visible);
 
+        case SET_PARTICIPANT_MENU_BUTTONS_WITH_NOTIFY_CLICK:
```

Comment:
Which one would you choose? 

This is used in the video menu of the participant, in participant pane and possibly somewhere else. Also the format of the data doesn't include participant id or anything...

In the same time buttonsWithNotifyClick also includes buttons not from the toolbox ( example the add password button ).

Originally I was also thinking to put it somewhere else but since I didn't find reducer that fits well participantMenuButtonsWithNotifyClick I decided to not overcomplicate the whole thing...

But if you have something better in mind I don't mind moving it.

Reply:
I agree, it's probably better to have the whole notify buttons thingie in the same place.